README.md

```diff
@@ -1,362 +1,71 @@
-
-  `Chain.get_blocks(start_block_hash, number_of_blocks)`
-
-- To get the longest chain:
-
-  `Chain.longest_blocks_chain()`
-
-##### Miner :
-
-- To get a candidate block:
-
-  `Miner.candidate()`
-
-- To get the current value of the coinbase:
-
-  `Miner.coinbase_transaction_value()`
-
-- To get the current state of the miner:
-
-  `Miner.get_state()`
-
-##### Peers:
-  The default sync port is 3015, it can be set manually by running the node with `SYNC_PORT=some_port iex -S mix`. This port is different from the one used by the phoenix server application.
-
-- To get all peers:
-
-  `Peers.all_peers()`
-
-- Connect to a peer by specifying the address (host, has to be in single quotes i.e. 'localhost'), port (SYNC_PORT) and peer pubkey (different from the keypair which is used for transaction signing):
-
-  `Peers.try_connect(%{host: host, port: port, pubkey: pubkey})`
-
-- Connecting to a peer can also be done by using `Peers.get_info_try_connect(uri)` which gets the peer info from a phoenix endpoint (peer that is being added needs to have the phoenix server running).
-
-##### Transaction Pool:
-
-- To inspect all transactions in the Transaction Pool:
-
-  `Pool.get_pool()`
+#### Starting the application
+Start the application in interactive Elixir
+- Development config: `make iex-0`
+- Production config: `MIX_ENV=prod make iex-0`
 
+The default sync port is 3015, this can be adjusted using `SYNC_PORT=some_port iex -S mix phx.server`.
+The node will run an http api at `localhost:4000`, this can be adjusted using `PORT=some_port iex -S mix phx.server`.
 
-### HTTP-API
-
-- The node will run an http API at: `localhost:4000`
-
-- To get the current info of the node:
-
-  `GET localhost:4000/info`
-
-- To get the peers your node is connected to:
-
-  `GET localhost:4000/peers`
-
-- To get all blocks from the current chain:
-
-  `GET localhost:4000/blocks`
-
-- To get all blocks with full information about the blocks:
-
-  `GET localhost:4000/raw_blocks`
-
-- To get the transactions in the Transaction Pool:
-
-  `GET localhost:4000/pool_txs`
-
-- To post new transaction to the Transaction Pool:
-
-  `POST localhost:4000/new_tx`
-
-  Body: **serialized_tx**
-
-  Where *serialized_tx* is json serialized signed transaction structure
-
-- To post new block to the chain:
-
-  `POST localhost:4000/block`
-
-  Body: **serialized_block**
-
-  Where *serialized_block* is a json serialized block structure
-
-- To get all transactions for an account
-
-  `GET localhost:4000/tx/{account}`
-
-  Where *account* is a hex encoded public key
-
-- To get a block by hash
-
-  `GET localhost:4000/block/{block_hash}`
+## Usage
 
-  Where *block_hash* is the bech32 encoded block header hash
+### Elixir interactive api-calls
+- `Miner.resume()` to start the miner
+- `Miner.suspend() ` to stop the miner
 
-- To get an account balance:
 
-  `GET localhost:4000/balance/{account}`
+- `Chain.top_block()` to add get the top block of the current chain
```

Comment:
Supposed to be: to get the top block of the current chain

Reply:
Maybe will be good to be add `Miner.mine_sync_block_to_chain` as well. What do you think?

Reply:
yes, will adjust and add this